### DISCUSSION TITLE
Minor issue (Missing character in a test function)

### DISCUSSION THREAD

**Discussion by OliverMensahDev** [composer-unused/composer-unused#224] [General]
I was working on  this issue #219  for hacktoberfest and I realized that a character was missing in this function `itShouldNoReportUnusedWithAutoloadFilesWithRequire`
\Icanhazstring\Composer\Test\Unused\Integration\UnusedCommandTest::itShouldNoReportUnusedWithAutoloadFilesWithRequire

The `t` in the `not` didn't show so I think this should be itShouldNotReportUnusedWithAutoloadFilesWithRequire.

**Comment by icanhazstring** [comment]
Hi. Yes sure. If you are working on the issue, just rename it 😉

Could catch though 👍
